### PR TITLE
Overflow issue in blog space

### PR DESCRIPTION
**Pull request by Adity20**
**What kind of change does this PR introduce?**
Handle Overflow in Blog Space

**Issue Number:**
-  Closes ##393  
-  Related to #393  

**Screenshots/videos:**

**Summary**
The overflow issue was causing some of the blog content to be hidden or cut off, making it difficult for users to read the entire blog post. By handling the overflow, we ensure that all content is visible and presented in a clean, organized manner.

@benjagm @DhairyaMajmudar kindly review this PR for issue #393

### CODE REVIEW

File: pages/draft/2020-12/release-notes.md [hunk 1]
```diff
@@ -155,60 +155,64 @@ Here's how you would covert a schema using `$recursiveRef` to use `$dynamicRef`.
...
-        {
-          "$schema": "https://json-schema.org/draft/2019-09/schema",
-          "$id": "https://example.com/strict-tree",
-          "$recursiveAnchor": true,
-          "$ref": "tree",
-          "unevaluatedProperties": false
-        }
-      ```
-    </td>
+
```
Comment (DhairyaMajmudar) [inline pages/draft/2020-12/release-notes.md:158]: Why doing changes with this file ?

File: pages/blog/index.page.tsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ export default function StaticMarkdownPage({ blogPosts }: { blogPosts: any[] })
       <Head>
         <title>JSON Schema Blog</title>
       </Head>
-      <div className='max-w-[1400px] mx-auto flex flex-col items-center mt-10'>
+      <div className='max-w-[1400px] mx-auto flex flex-col items-center mt-10 overflow-hidden'>
```
Comment (DhairyaMajmudar) [inline pages/blog/index.page.tsx:71]: Can you pls. make it overflow hidden in x direction only. using overflow-x-hidden

**Review by DhairyaMajmudar — CHANGES_REQUESTED** [review]

**Comment by DhairyaMajmudar** [comment]
Thanks @Adity20 for PR , just left some comments pls. have a look : )

File: pages/draft/2020-12/release-notes.md (hunk 1 above)
Comment (Adity20) [reply pages/draft/2020-12/release-notes.md:158]: These changes are for another issue.

File: pages/blog/index.page.tsx (hunk 2 above)
Comment (Adity20) [reply pages/blog/index.page.tsx:71]: okay @DhairyaMajmudar

**Comment by Adity20** [comment]
Now check @DhairyaMajmudar

File: pages/draft/2020-12/release-notes.md (hunk 1 above)
Comment (Adity20) [reply pages/draft/2020-12/release-notes.md:158]: how can i remove the changes of this file and keep the ones for this issue only in this pull request

**Comment by Adity20** [comment]
I'll create another PR there are some unresolved conflicts

**Comment by benjagm** [comment]
Thanks @Adity20 !

**Pull request closed without merging**